### PR TITLE
ZkVM: addressing spec feedback

### LINKED ISSUE #71: ZkVM: Spec Questions
https://github.com/interstellar/slingshot/blob/main/zkvm/spec/ZkVM.md#aggregated-signature

"Commit all verification keys P[i] in order, one by one"

Would this be in lexographical order?

Can the terms clear-text scalars and non-secret scalars be used interchangeably?

The term "Flavour" was defined in the Cloak protocol, should we copy over the definition for the specs?

### PR DESCRIPTION
**Pull request by oleganza**
Clarifying the spec. 
Closes #71.

### CODE REVIEW

File: zkvm/spec/ZkVM.md [hunk 1]
```diff
@@ -737,12 +737,12 @@ Aggregated Signature is encoded as a 64-byte [data](#data-type).
...
 The protocol is the following:
 
-1. Prover and verifier obtain a [transcript](#transcript) `T` defined by the context in which the signature is used (see [`signtx`](#signtx), [`delegate`](#delegate)). The transcript is assumed to be already bound to the _message_ being signed.
+1. Prover and verifier obtain a [transcript](#transcript) `T` that is assumed to be already bound to the _message_ being signed (see [`signtx`](#signtx) and [transaction signature](#transaction-signature)).
 2. Commit the count `n` of verification keys as [LE32](#le32):
     ```
     T.commit("n", LE32(n))
     ```
-3. Commit all verification keys `P[i]` in order, one by one:
+3. Commit all verification keys `P[i]` one by one (in order they were collected):
```
Comment (vickiniu) [inline zkvm/spec/ZkVM.md]: nit: this phrasing still seems a little unclear to me. Maybe "one by one (in the order they were added in the program)" ?

Comment (oleganza) [reply zkvm/spec/ZkVM.md]: image

how about this wording?

Comment (vickiniu) [reply zkvm/spec/ZkVM.md]: great!!!!
